src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/TestLeoRoutes.scala

```diff
@@ -12,7 +12,7 @@ import scala.concurrent.duration._
   */
 trait TestLeoRoutes { this: ScalatestRouteTest =>
   val mockGoogleDataprocDAO = new MockGoogleDataprocDAO
-  val leonardoService = new LeonardoService(mockGoogleDataprocDAO, DbSingleton.ref)
+  val leonardoService = new LeonardoService(mockGoogleDataprocDAO, DbSingleton.ref, system.deadLetters)
```

Comment:
?

Reply:
The constructor needs an actor, but the route tests don't actually use it. This was a way of passing in a "no-op" actor.